bin/CMakeLists.txt

```
@@ -0,0 +1,14 @@
+# -*- cmake -*-
+#
+# michael a.g. aïvázis
+# orthologue
+# (c) 1998-2019 all rights reserved
+#
+
+# copy all the configuration files to the install directory
+install(
+  FILES pyre pyre-config merlin smith.pyre
```

```suggestion
  PROGRAMS pyre pyre-config merlin smith.pyre
```
since these are executable scripts